src/components/projects/projectId/data-browser/DataBrowser.tsx

```diff
@@ -64,7 +56,7 @@ export default function DataBrowser() {
         if (!projectId) return;
         if (!searchRequest) return;
         if (searchRequest.offset == 0) return;
-        refetchExtendedRecord({ variables: { projectId: projectId, filterData: JSON.stringify({}), offset: searchRequest.offset, limit: searchRequest.limit } }).then((res) => {
+        searchRecordsExtended(projectId, [], searchRequest.offset, searchRequest.limit, (res) => {
```

Comment:
just a note, there is a change in the request here in one other PR for sending the data if there are some conflicts it needs to be changed